plugins/carbon_black_response/plugin.spec.yaml

```diff
@@ -737,6 +774,7 @@ actions:
         description: The lists of alerts
```

Comment:
Include a title in every input and output for every action & trigger throughout the plugin